features/support/hooks.rb

```diff
@@ -19,6 +19,45 @@
     Howitzer::Log.info 'IE reset session'
     Capybara.current_session.execute_script("void(document.execCommand('ClearAuthenticationCache', false));")
   end
+
+  test_teardown = Howitzer::Cache.extract(:teardown)
```

Comment:
I have some doubts about removing test data in teardown in general. It is not 100% clean solution. 
Right now I would keep it as it is, but implementation should be rewritten. Main problem here - removing test data via UI. It is really really bad idea. Note, we create data via API and MUST destroy data via API. After quick investigation I found we have everything implemented already. Just take a look here https://github.com/strongqa/demo_web_app/blob/master/app/controllers/api/v1/categories_controller.rb#L36
and gem which we use
https://github.com/balvig/spyke

So, I expect it should work:

```ruby
c = create(:category)
c.destroy
```

Reply:
Storing full objects in memory is a bit wrong approach as well, cause it will consume additional memory.
In fact, we need to keep model name and id only. Then we could do in teardown:

```ruby
Article.find(5).destroy
```

Reply:
fixed